src/main/resources/com/chikli/hudson/plugin/naginator/Messages.properties

```diff
@@ -1,2 +1,9 @@
 NaginatorCause.Description=Started by Naginator after build {0} failure
+RegexpForMatrixStrategy.TestChildrenRetriggerAll=Test failed matrix parts, and retrigger all
```

Comment:
So thats the case I described in the ticket I guess.
If so, im fine with the messages